src/canvas.cpp

```diff
@@ -272,6 +290,34 @@ void Canvas::paintGL()
     painter.drawText(10, height() - textHeight, status);
 }
 
+/*
+// 1. Создание VAO/VBO для квада (однократно)
+void testquad() {
+GLuint quadVAO, quadVBO;
+float quadVertices[] = {
+    // positions     // colors
+    -1000.0f, -1000.0f, 0.0f,  1.0f, 0.0f, 0.0f, 0.5f,
+     1000.0f, -1000.0f, 0.0f,  1.0f, 0.0f, 0.0f, 0.5f,
+     1000.0f,  1000.0f, 0.0f,  1.0f, 0.0f, 0.0f, 0.5f,
+    -1000.0f,  1000.0f, 0.0f,  1.0f, 0.0f, 0.0f, 0.5f
+};
+
+glGenVertexArrays(1, &quadVAO);
+glGenBuffers(1, &quadVBO);
+glBindVertexArray(quadVAO);
+glBindBuffer(GL_ARRAY_BUFFER, quadVBO);
+glBufferData(GL_ARRAY_BUFFER, sizeof(quadVertices), quadVertices, GL_STATIC_DRAW);
+glVertexAttribPointer(0, 2, GL_FLOAT, GL_FALSE, 6 * sizeof(float), (void*)0);
+glEnableVertexAttribArray(0);
+glVertexAttribPointer(1, 4, GL_FLOAT, GL_FALSE, 6 * sizeof(float), (void*)(2 * sizeof(float)));
+glEnableVertexAttribArray(1);
+
+// 2. Рендеринг в основном цикле
+simpleShader.bind();
+glBindVertexArray(quadVAO);
+glDrawArrays(GL_TRIANGLE_FAN, 0, 4);
+}
+*/
```

Comment:
Please don't commit commented-out code blocks. I assume this was used as a reference during development, but it shouldn't stay here.